ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1608,12 +1610,21 @@ def test_ipa_dns_systemrecords_check(self):
         Test checks the result of IPADNSSystemRecordsCheck
         when ipa-server is configured without DNS.
         """
-        expected_msgs = {
-            "Expected SRV record missing",
-            "Got {count} ipa-ca A records, expected {expected}",
-            "Got {count} ipa-ca AAAA records, expected {expected}",
-            "Expected URI record missing",
-        }
+        version = tasks.get_healthcheck_version(self.master)
+        if (parse_version(version) < parse_version('0.12')):
+            expected_msgs = {
+                "Expected SRV record missing",
+                "Got {count} ipa-ca A records, expected {expected}",
+                "Got {count} ipa-ca AAAA records, expected {expected}",
+                "Expected URI record missing",
+            }
+        else:
+            expected_msgs = {
+                "Expected SRV record missing",
+                "Unxpected ipa-ca address {server}",
```

Comment:
There is a typo and the exact message should be the same as https://github.com/freeipa/freeipa-healthcheck/blob/master/src/ipahealthcheck/ipa/idns.py#L219
```
Unexpected ipa-ca address {ipaddr}
```

Reply:
Sigh. I really need to be more careful. Fixed.